### PR TITLE
Add redirects for 404 URLs

### PR DESCRIPTION
**Pull request by kickoke**
A content audit revealed that we have 3 URLs that return 404 status codes. I've set up redirects to address the issue.

### CODE REVIEW

**Review by bwklein — APPROVED** [review]

**Review by bwklein — CHANGES_REQUESTED** [review]
Looks like the build is breaking with this config file.

File: netlify.toml [hunk 1]
```diff
@@ -21,3 +21,14 @@ HUGO_VERSION = "0.81.0"
 
 [context.next.environment]
 HUGO_ENABLEGITINFO = "true"
+
+[[redirects]]
+
+from = "/docs/glossaries/contributing/roadmap"
+to = "/docs/glossaries/roadmap/"
+
```
Comment (bwklein) [inline netlify.toml:29]:
```suggestion
[[redirects]]

```

File: netlify.toml [hunk 2]
```diff
@@ -21,3 +21,14 @@ HUGO_VERSION = "0.81.0"
...
+from = "/docs/glossaries/primer/maturity-levels"
+to = "/docs/glossaries/maturity-levels/"
+
```
Comment (bwklein) [inline netlify.toml:32]:
```suggestion

[[redirects]]
```

File: netlify.toml [hunk 3]
```diff
@@ -21,3 +21,16 @@ HUGO_VERSION = "0.81.0"
 
 [context.next.environment]
 HUGO_ENABLEGITINFO = "true"
+
+[[redirects]]
+
+from = "/docs/glossaries/contributing/roadmap"
+to = "/docs/glossaries/roadmap/"
+[[redirects]]
```
Comment (bwklein) [inline netlify.toml:29]:
```suggestion

[[redirects]]
```

File: netlify.toml [hunk 4]
```diff
@@ -21,3 +21,16 @@ HUGO_VERSION = "0.81.0"
...
+
+from = "/docs/glossaries/primer/maturity-levels"
+to = "/docs/glossaries/maturity-levels/"
+
+[[redirects]]
```
Comment (bwklein) [inline netlify.toml:34]:
```suggestion
[[redirects]]

```

**Review by bwklein — APPROVED** [review]

**Comment by kickoke** [comment]
Oh, I wasn't aware that each section needed the redirects tag. Thanks Bryan! :)

**Pull request merged**